publish/canisters/BUILD.bazel

```diff
@@ -51,25 +52,30 @@ COMPRESSED_CANISTERS = {
 
 DEFAULT_CANISTERS_MAX_SIZE_E5_BYTES = "21"
 
-CANISTERS_MAX_SIZE_E5_BYTES = {
-    "ic-ckbtc-minter.wasm": "19",
-    "ledger-canister.wasm": "19",
-    "ledger-canister_notify-method.wasm": "19",
-}
-
 CANISTERS_MAX_SIZE_COMPRESSED_E5_BYTES = {
     # -- FI team --
     # The compressed version of these two canisters should be ~600kb,
     # we are setting the check to 7 to leave some space for growth
     # but enough to get an alert in case of a spike in size.
     "ic-icrc1-ledger.wasm.gz": "7",
     "ic-icrc1-ledger-u256.wasm.gz": "7",
+    # Size when constraint addded: 841_234 bytes
+    "ledger-canister.wasm.gz": "9",
+    # Size when constraint addded: 847_186 bytes
+    "ledger-canister_notify-method.wasm.gz": "9",
```

Comment:
@mbjorkqvist : Feel free to let me know if you would like other bounds.

Reply:
That LGTM, thanks @gregorydemay !